doc/source/ray-core/scheduling/resources.rst

```diff
@@ -176,3 +182,7 @@ The precision of the fractional resource requirement is 0.0001 so you should avo
 
   Besides resource requirements, you can also specify an environment for a task or actor to run in,
   which can include Python packages, local files, environment variables, and more---see :ref:`Runtime Environments <runtime-environments>` for details.
+
+Dynamic Resource Update
+-----------------------
+Ray **does not permit dynamic updates of resource capacities after a node has begun**.
```

Comment:
This should just be a warning in the above (like line 66), not a separate section since it looks a bit more weird

Reply:
addressed